utilities/transactions/transaction_base.cc

```diff
@@ -334,9 +355,12 @@ Status TransactionBaseImpl::Put(ColumnFamilyHandle* column_family,
 }
 
 Status TransactionBaseImpl::Merge(ColumnFamilyHandle* column_family,
-                                  const Slice& key, const Slice& value) {
+                                  const Slice& key, const Slice& value,
+                                  const bool assume_exclusive_tracked) {
+  const bool do_validate = !assume_exclusive_tracked;
```

Comment:
Can we rename to just assume_tracked, if we agree that shared locks also imply that we can skip validation?